UnitsNet/GeneratedCode/UnitSystem.Default.g.cs

```diff
@@ -1731,6 +1731,90 @@ public sealed partial class UnitSystem
                                 new AbbreviationsForCulture("en-US", "THz"),
                             }),
                     }),
+                new UnitLocalization(typeof (HeatFluxUnit),
+                    new[]
+                    {
+                        new CulturesForEnumValue((int) HeatFluxUnit.BtuPerHourSquareFoot,
+                            new[]
+                            {
+                                new AbbreviationsForCulture("en-US", "BTU/h·ft²"),
+                            }),
+                        new CulturesForEnumValue((int) HeatFluxUnit.BtuPerMinuteSquareFoot,
+                            new[]
+                            {
+                                new AbbreviationsForCulture("en-US", "BTU/min·ft²"),
+                            }),
+                        new CulturesForEnumValue((int) HeatFluxUnit.BtuPerSecondSquareFoot,
+                            new[]
+                            {
+                                new AbbreviationsForCulture("en-US", "BTU/s·ft²"),
```

Comment:
We need to nail this form first. I personally find it ambiguous the way it reads now, but I am no domain expert on this.

A quick [google](https://www.google.no/search?q=BTU%2Fs%C2%B7ft%C2%B2&oq=BTU%2Fs%C2%B7ft%C2%B2&aqs=chrome..69i57j69i58.167j0j7&sourceid=chrome&ie=UTF-8) tells me that the these forms are used: 
* `Btu (IT)/s/ft²`
* `Btu/(s.ft2)`
* `Btu (IT)/second/ft²`
* `Btuth·in/(h·ft2·°F)` (just consider the form here)

So from this, I conclude that we need to use parantheses or use a second division. What do you think?

Reply:
I can point you to one of my company's product offerings, ANSYS Mechanical. It is widely used by engineers and uses the BTU/s·in² format. I again expect this due to order of operations and it's the way I am used to seeing heat flux expressed.

![](https://i1.wp.com/www.padtinc.com/blog/wp-content/uploads/2015/01/thermal-submodeling-3.jpg)

Reply:
Alright, I'm sold :-) We will keep this form then since at least in the context of these units they seem commonly used.

Reply:
Sounds good.